packages/cli/src/commands/generate.ts

```diff
@@ -0,0 +1,69 @@
+import { Command, Option } from 'clipanion';
+import { cp, readFile, writeFile, rename } from 'node:fs/promises';
+import { dirname, join } from 'node:path';
+import { spawnSync } from 'node:child_process';
+import { pascalCase } from 'change-case';
+import { BaseCommand } from './base';
+
+export class GenerateCommand extends BaseCommand {
+  static override paths = [['generate'], ['gen']];
+
+  static override usage = Command.Usage({
+    // category: `My category`,
+    description: `Generate a boilerplated project for a custom \`ConfigStore\``,
+    // details: `
+    //   A longer description of the command with some \`markdown code\`.
+
+    //   Multiple paragraphs are allowed. Clipanion will take care of both reindenting the content and wrapping the paragraphs as needed.
+    // `,
+    // examples: [
+    //   [`A basic example`, `$0 my-command`],
+    //   [`A second example`, `$0 my-command --with-parameter`],
+    // ],
+  });
+
+  directory = Option.String('--dir,--di', {
+    description: `Location to generate the \`ConfigStore\` project`,
+    required: true,
+  });
+
+  name = Option.String('--name,--na', {
+    description: `Name of the \`ConfigStore\``,
+    required: true,
+  });
+
+  async execute() {
+    await this.init();
+
+    console.log(`Copying from ${join(dirname(import.meta.dirname), 'integration_template')} to ${this.directory}`);
```

Comment:
Happy to change any of the `console.log` contents here, but figured it'd be nice to have something there to give the user some sense of progress.

Reply:
use the this.stdio (based on the `consola` dependency)